src/main/java/com/example/kanbanserv/domain/Project.java

```diff
@@ -1,4 +1,95 @@
 package com.example.kanbanserv.domain;
 
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
 public class Project {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String projectIdentifier;
+    private String projectName;
+    private String description;
+    private Date start_date;
+    private Date end_date;
+    private Date created_At;
+    private Date updated_At;
+
+    public Project(){
+
+    }
+    public Long getId() {
```

Comment:
Nit: add one line break in between

Reply:
Have added @Data annotation for all getters and setters. Hence, removing this.

Reply:
cool